### PR TITLE
Fixes #36756 - Add a banner warning about RHEL lifecycle EOS per content host

### PR DESCRIPTION
**Pull request by lfu**
#### What are the testing steps for this pull request?
A banner is displayed warning about RHEL lifecycle EOS per content host

### CODE REVIEW

**Comment by lfu** [comment]
Screenshot at 2023-09-20 16-28-33

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/hosts/host.factory.js [hunk 1]
```diff
@@ -30,6 +30,10 @@ angular.module('Bastion.hosts').factory('Host',
             return _.isString(this.operatingsystem_name) && (this.operatingsystem_name.indexOf("Debian") >= 0 || this.operatingsystem_name.indexOf("Ubuntu") >= 0);
         };
 
+        resource.prototype.rhelLifecycleExpireSoon = function() {
```
Comment (chris1984) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/hosts/host.factory.js:33]: Do you like this format better or the React way of doing things?

Comment (lfu) [reply engines/bastion_katello/app/assets/javascripts/bastion_katello/hosts/host.factory.js:33]: React is easier to understand to me.

**Review by chris1984 — COMMENTED** [review]
I like the latest changes, nice work Lucy :smile: Will test it with a few RHEL hosts Monday. Code wise it looks great

**Comment by chris1984** [comment]
ACK works well, tested with RHEL 6/7/8

Screenshot 2023-10-02 at 14-49-17 Content Host rhel6 area51 example com
Screenshot 2023-10-02 at 14-23-48 Content Host rhel8 area51 example com
Screenshot 2023-10-02 at 14-21-21 Content Host rhel7 area51 example com

**Review by chris1984 — APPROVED** [review]

**Pull request merged**